recipes-support/python-cryptoauthlib/python3-cryptoauthlib_20200208.bb

```diff
@@ -14,4 +14,6 @@ inherit pypi setuptools3
 
 RDEPENDS_${PN} += "python3-core python3-cryptography python3-ctypes python3-datetime python3-netclient"
 
-DEPENDS += "cmake-native"
+DEPENDS += "cmake-native udev"
```

Comment:
Thanks @hsnhrn ! I have one more thing to ask. Could you please replace "udev" here with ${@bb.utils.contains('DISTRO_FEATURES','systemd','udev','eudev',d)} ? This will assure the dependency is accomplished also for images without systemd. Thanks!

Reply:
I does, both eudev and systemd provide udev. However, eudev most not be used if systemd is present: https://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/recipes-core/udev/eudev_3.2.9.bb?h=dunfell#n28 
Though looking at it better now, I think it was OK in the first place, using "udev" alone. The system itself will choose the "udev" provider, based on systemd availability.  @hsnhrn do you mind switching back to the first version of the pull-request? Sorry about this, you were right in the first place

Reply:
yeah sure! :+1: